lib/core/src/display/shape/glyph/system.rs

```diff
@@ -0,0 +1,231 @@
+//! A main glyph system implementation.
+
+use crate::prelude::*;
+
+use crate::display::shape::glyph::font::FontId;
+use crate::display::shape::glyph::font::FontRenderInfo;
+use crate::display::shape::glyph::font::FontRegistry;
+use crate::display::shape::glyph::pen::PenIterator;
+use crate::display::shape::glyph::msdf::MsdfTexture;
+use crate::display::symbol::material::Material;
+use crate::display::symbol::shader::builder::CodeTemplate;
+use crate::display::world::*;
+use crate::system::gpu::data::texture::*;
```

Comment:
you can use `use crate::system::gpu::texture::*;` instead :)

Reply:
And that's it. I have no idea what is the import I'm supposed to use. For me is much simpler to use direct imports, for both reading code as well as writing (where IDE can do all the Job for me)